tasks.py

```diff
@@ -375,6 +375,14 @@ def test_module(module=None, verbosity=2):
 
 
 @task
+def jshint():
+    """Run the jshint."""
+    for dirpath, dirnames, filenames in os.walk("."):
```

Comment:
If I run jshint with a directory it will just hang and I'll have to kill the process, unless I'm more specific with what directory. For example  `jshint node_modules/` hangs but `jshint node_modules/async/` works even though the .js file in a child directory. Maybe it's faster to do it iteratively?